docs/fleet/machines.md

```diff
@@ -92,6 +92,11 @@ For local communication between [parts](/build/configure/parts-and-remotes/#mach
 <br>
 
 You can also access the control interface using the [Viam mobile app](/fleet/#the-viam-mobile-app), which you can find on the [App Store](https://apps.apple.com/vn/app/viam-robotics/id6451424162) and on [Google Play](https://play.google.com/store/apps/details?id=com.viam.viammobile&hl=en&gl=US).
+The mobile app's control interface includes functionality to access your machine logs, as well as upload images directly from your phone.
+You can access uploaded images on the Viam app's [**DATA** tab](https://app.viam.com/data/) for later use.
+See [Upload images with the Viam mobile app](https://docs.viam.com/data/upload/#upload-images-with-the-viam-mobile-app) for more information.
```

Comment:
I agree with Jessamy in that this section is focused on the Control features in the App, and likely doesn't need to address the non-control features it offers, like data upload and machine log review. In fact, the link to `Viam mobile app` in the preceding sentence takes us to a paragraph that introduces both.

I like that you added some value-add content in your original copy, `organized by location and organziation` and `search for machines in your fleet` -- both things I would find very useful to do from my phone were I in charge of a large warehouse fleet!

So maybe, combining Jessamy's feedback with your original copy, I might do something like this:

```suggestion
The Viam mobile app gives you the ability to search through the machines in your fleet, ordered by location, and control the specific machine or machine component that you need.
You can watch the live camera feed, adjust the component or machine's runtime parameters, and switch between controllable components just as you would in the Viam app. 
```

(This is where I'd put a `For more information, see [Viam mobile app]`, but we have that already in the preceding sentence, so no need!)

Whatcha think?